static/src/deploys-radiator/app/main.ts

```diff
@@ -57,7 +57,7 @@ const run = (): Promise<void> => {
             .sortBy(deploy => deploy.build)
             .first();
 
-        return [ latestCodeDeploy, oldestProdDeploy ];
```

Comment:
Here I have added the return type as a type assertion. TypeScript uses arrays as tuples: without the type assertion it would think this is a type of `Array<DeployRecord>` instead of `[ DeployRecord, DeployRecord ]` (tuple).